tee-worker/ts-tests/bulk_identity.test.ts

```diff
@@ -36,36 +38,36 @@ describeLitentry('multiple accounts test', 10, async (context) => {
     });
     step('send test token to each account', async () => {
         const txs: any = [];
-        for (let i = 0; i < substraetSigners.length; i++) {
+        for (let i = 0; i < substrateSigners.length; i++) {
             //1 token
-            const tx = context.api.tx.balances.transfer(substraetSigners[i].address, '1000000000000');
+            const tx = context.api.tx.balances.transfer(substrateSigners[i].address, '1000000000000');
             txs.push(tx);
         }
-        await context.api.tx.utility.batch(txs).signAndSend(context.substrateWallet.alice);
-        await listenEvent(context.api, 'balances', ['Transfer'], txs.length, [
-            u8aToHex(context.substrateWallet.alice.addressRaw),
-        ]);
+
+        await new Promise((resolve) => {
+            context.api.tx.utility
+                .batch(txs)
+                .signAndSend(context.substrateWallet.alice, async (result: SubmittableResult) => {
+                    if (result.status.isInBlock) {
+                        console.log(`Transaction included at blockHash ${result.status.asInBlock}`);
+                        resolve(result.status);
+                    } else if (result.status.isInvalid) {
+                        console.log(`Transaction is ${result.status}`);
+                    }
+                });
+        });
```

Comment:
Do we want to:
- `resolve` when `result.status.isFinalized`
- `reject` when `result.status.isInvalid`

? :thinking: 

Compare `signAndSend` at `ts-tests/tests/utils.ts`

Reply:
Fixed.

Reply:
Here's a special case where [`disable_vc` and `revoke_vc`](https://github.com/litentry/litentry-parachain/blob/ad3575322a31aba111588a2596519511538b218f/pallets/vc-management/src/lib.rs#L235) happen instantaneously and are modified directly in the parachain without going through the TEE. Therefore, if we wait for `result.status.isFinalized` before listening, it's possible that the block containing the changes has already passed. Thus, we need to start listening at `result.status.asInBlock`.